h2/src/docsrc/html/features.html

```diff
@@ -950,7 +950,12 @@ <h3>PostgreSQL Compatibility Mode</h3>
 </li><li>The system columns <code>CTID</code> and
     <code>OID</code> are supported.
 </li><li>LOG(x) is base 10 in this mode.
-</li><li>REGEXP_REPLACE() uses \ for back-references.
+</li><li>REGEXP_REPLACE():
+    <ul>
+    <li>Uses \ for back-references</li>
+    <li>Will not throw an exception when the <code>flagsString</code> parameter contains a 'g'</li>
+    <li>Will use Java's <code>String.replaceFirst()</code> method (instead of <code>String.replaceAll()</code>) if the 'g' flag is not present in the <code>flagsString</code> parameter.</li>
```

Comment:
Implementation's details are not relevant here, may be just notice that in absence of 'g' flag only the first matched substring will be replaced?